src/exe/cimbar_send/send.cpp

```diff
@@ -76,21 +69,22 @@ int main(int argc, char** argv)
 	bool running = true;
 	bool start = true;
 
-	auto draw = [&w, &cam, use_rotatecam, delay, &running, &start] (const cv::Mat& frame, unsigned) {
+	auto draw = [&w, use_rotatecam, use_shakycam, delay, &running, &start] (const cv::Mat& frame, unsigned) {
 		if (!start and w.should_close())
 			return running = false;
 		start = false;
 
-		cv::Mat& windowImg = cam.draw(frame);
-		w.show(windowImg, delay);
+		w.show(frame, delay);
 		if (use_rotatecam)
 			w.rotate();
+		if (use_shakycam)
+			w.shake();
 		return true;
 	};
 
 	Encoder en(ecc, cimbar::Config::symbol_bits(), colorBits);
 	while (running)
 		for (const string& f : infiles)
-			en.encode_fountain(f, draw, compressionLevel);
+			en.encode_fountain(f, draw, compressionLevel, 8.0, window_size);
```

Comment:
I may go a different direction with this at some point. We're passing in the size here to embed the 1024x1024 image in a larger image -- which will then get passed to the cimbar::window (which won't have to worry about resizing it).

This is somewhat silly -- the window should be able to render an image on a larger background.

But you know what else is silly? Trying to get transpose and scaling to work in a GLSL shader without hitting weird half-pixel blurring issues.

Reply:
https://github.com/sz3/libcimbar/pull/54/commits/a2ffe08ccdbfae654191c28763dc661f66bc88c2